tests/wasm/api.js

```diff
@@ -55,3 +55,7 @@ exports.debug_values = () => ([
     () => (null),
     new Set(),
 ]);
+
+exports.assert_function_table = x => {
```

Comment:
Maybe it makes sense to assert that this is the right function table too?

Imagining something like this:

```rust
assert_function_table(wasm_bindgen::function_table(), my_exported_function as *const _ as usize);
```

```js
exports.assert_function_table = (t, i) => {
    assert.ok(t instanceof WebAssembly.Table);
    assert.equal(t.get(i), wasm.my_exported_function);
};
```

Reply:
Sounds like a great idea to me!